src/components/Modal/Example.md

```diff
@@ -283,3 +283,51 @@ class ModalConsumer extends React.Component {
 
 <ModalConsumer />
 ```
+
+**Prompting on before closing**
+
+Modal with custom confirmation on close.
+
+```jsx
```

Comment:
We can skip it as a dummy snapshot, which doesn't bring any value
```suggestion
```jsx { "props": { "data-skip": true } }
```